Ginger/Ginger/WizardLib/WizardWindow.xaml.cs

```diff
@@ -38,12 +38,14 @@ public partial class WizardWindow : Window, IWizardWindow
         public static WizardWindow CurrentWizardWindow = null;
 
         WizardBase mWizard;
+        static bool mIsPrevButtonDisable = false;
 
         List<ValidationError> mValidationErrors = new List<ValidationError>();
 
-        public static void ShowWizard(WizardBase wizard, double width = 800, double height = 800, bool DoNotShowAsDialog = false)
+        public static void ShowWizard(WizardBase wizard, double width = 800, double height = 800, bool DoNotShowAsDialog = false, bool IsPrevButtonDisable = false)
         {
```

Comment:
instead of adding param to ShowWizard() func, just expose the PrevButtonVisability from the window like have 
Public Visibality PrevBtnVisability
{ get{ return xPrevButton.Visability;} set {PrevButton.Visability;=value ;}

or create Public function called HidePrevBtn() which will make it collapsed and call the fuinction from shared repo wizard

then from shared Repo Wizard, make it Collapsed so it won't show in case of shared repo wizard

Reply:
Ok Meni.. I will do it

Reply:
let me know when it ready to be reviewed

Reply:
Ok...I am done with requested changes...we will review it tomorrow